packages/api/src/task/scheduler.ts

```diff
@@ -285,11 +292,11 @@ export default class TaskScheduler {
       ...updates,
     };
     const timestamp = asset.status.updatedAt;
-    await this.queue.publishWebhook("events.asset.status", {
+    await this.queue.publishWebhook("events.asset.updated", {
```

Comment:
We should remove the `if (!updates.status || updates.status === asset.status) {` above since we should always be sending these updates. Not only when there is a status change.

Or rather... Maybe we should be moving this `publishWebhook` up, to before the `if` check. The other webhooks below we still want to fire only when the status does change.